src/tye/InitHost.cs

```diff
@@ -105,5 +110,49 @@ private static void ThrowIfTyeFilePresent(FileInfo? path, string yml)
                 throw new CommandException($"File '{tyeYaml}' already exists. Use --force to override the {yml} file if desired.");
             }
         }
+
+        private static void AddToSolutionItems(FileInfo? path)
+        {
+            var slnFiles = Directory.GetFiles(path!.DirectoryName, "*.sln");
+            if (slnFiles.Length == 1)
+            {
+                var sln = slnFiles[0];
+
+                var lines = File.ReadAllLines(sln).ToList<string>();
+                var insertAt = 0;
+                var exists = false;
+
+                lines.ForEach(l =>
+                {
+                    if (l.IndexOf("(SolutionItems)") > 0)
```

Comment:
This logic doesn't work, `(SolutionItems)` may be specified in other itemgroups. For example in gRPC projects, often protobuf files are included as solution items under `Proto` directory e.g. [here](https://github.com/grpc/grpc-dotnet/blob/master/Grpc.DotNet.sln#L20-L35), this logic will add tye.yaml to that directory which wouldn't make any sense. You'll need to look for a project with the name `Solution Items`.